packages/client/components/ReflectionGroup/RemoteReflection.tsx

```diff
@@ -17,20 +17,16 @@ import getBBox from '../RetroReflectPhase/getBBox'
 import UserDraggingHeader, {RemoteReflectionArrow} from '../UserDraggingHeader'
 import useSpotlightResults from '~/hooks/useSpotlightResults'
 
-const circleAnimation = (transform?: string) => keyframes`
-  0%{
-    transform:translate(5px)
-              rotate(0deg)
-              translate(-5px)
-              rotate(0deg)
-              ${transform ?? ''}
-  }
-  100%{
-    transform:translate(5px)
-              rotate(360deg)
-              translate(-5px)
-              rotate(-360deg)
-              ${transform ?? ''}
+// original from https://codepen.io/Maelig/pen/KrjAgq
+// scale(0.75)
+// translate(-39.5px, 1px)
+// transform determined by hand by delta(kanban el, floating el)
+// round all decimals, convert to relative
+const triquetaPath = `path('m0 0c0 10-8 18-18 18s-18-8-18-18c3-2 6-2 9-2c10 0 18 8 18 18c0 6-3 12-9 16c-6-3-9-9-9-16c0-10 8-18 18-18c3 0 6 0 9 2z')`
```

Comment:
The animation moves down and to the left of the reflection's original position in the kanban. I think that works well, but it can hide some of the reflection below. 

Perhaps the reflection's original kanban position could be the centre of the animation rather than at the top right, i.e. the animation moves up and to the right initially before moving down and to the left. That way, it hides less off the card below. 

![wiggling](https://user-images.githubusercontent.com/39854876/142609605-df547499-c450-4db0-a3ba-8aa8ff7b52fa.gif)